### DISCUSSION TITLE
Can we replace `tabulate` with `prettytable`?

### DISCUSSION THREAD

**Discussion by opotowsky** [terrapower/armi#1747] [Q&A] [answered]
`tabulate` is a dead project and `prettytable` is not. But it's pretty extensively used here and downstream with the `runLog` module.

**Comment by john-science** [comment]
I would very much like to abandon this dead project as a dependency.  I'm game!

**Reply by opotowsky** [reply, marked as answer]
I'll convert this to an issue!

**Comment by opotowsky** [comment]
Closing, since it's now an issue
